modulemd/tests/test-version.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/bash
+
+set -x
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+pushd $MESON_BUILD_ROOT
+
+function error_out {
+    local code message
+    local "${@}"
+
+    echo $message 1>&2
+    exit $code
+}
+
+function common_finalize {
+    exitcode=$?
+
+    return $exitcode
+}
+
+trap common_finalize EXIT
+
+jq --version >/dev/null || error_out code=127 message="Install 'jq' to use this script"
```

Comment:
As you've probably noticed already, `jq` needs to be installed during the container build process. And probably added to the SPEC file.

Reply:
Yeah, saw that in the CI logs. Oops.

Reply:
It doesn't need to be added to the spec file because that builds with `-Ddeveloper_build=false`, suppressing this test.

Reply:
Ah. I overlooked the non-developer build from the SPEC.